apps/dashboard/src/components/shared/DeprecatedAlert.tsx

```diff
@@ -6,36 +6,38 @@ import {
   Flex,
 } from "@chakra-ui/react";
 import {
-  ArbitrumGoerli,
-  ArbitrumSepolia,
-  BaseGoerli,
-  BaseSepoliaTestnet,
   type Chain,
-  Goerli,
-  Mumbai,
-  OpSepoliaTestnet,
-  OptimismGoerli,
-  PolygonAmoyTestnet,
-  Sepolia,
-} from "@thirdweb-dev/chains";
+  type ChainMetadata,
+  arbitrumSepolia,
+  baseSepolia,
+  mumbai,
+  optimismSepolia,
+  polygonAmoy,
+  sepolia,
+} from "thirdweb/chains";
 import { Link, Text } from "tw-components";
 
 interface DeprecatedAlertProps {
-  chain: Chain | undefined;
+  chain: ChainMetadata | undefined;
 }
 
 const TO_BE_DEPRECATED_CHAINS: Record<number, { date: Date }> = {
-  [Mumbai.chainId]: {
+  [mumbai.id]: {
     date: new Date("2024-04-08"),
   },
 };
 
 const RECOMMENDED_CHAINS: Record<number, Chain> = {
-  [Goerli.chainId]: Sepolia,
-  [BaseGoerli.chainId]: BaseSepoliaTestnet,
-  [OptimismGoerli.chainId]: OpSepoliaTestnet,
-  [ArbitrumGoerli.chainId]: ArbitrumSepolia,
-  [Mumbai.chainId]: PolygonAmoyTestnet,
+  // ethereum goerli
```

Comment:
nit: comments say goerli but these are sepolia

Reply:
Yeah we are mapping the Goerli chain to the Sepolia ones. This component is for showing users alert that the chain they are on (Goerli-ish) is now deprecated and they should switch to the Sepolia counterpart.